tests/py2/test_ceil_floor.py

```diff
@@ -0,0 +1,48 @@
+# coding=utf-8
+#
+# This file is part of Hypothesis, which may be found at
+# https://github.com/HypothesisWorks/hypothesis-python
+#
+# Most of this work is copyright (C) 2013-2017 David R. MacIver
+# (david@drmaciver.com), but it contains contributions by others. See
+# CONTRIBUTING.rst for a full list of people who may hold copyright, and
+# consult the git log if you need to determine who owns an individual
+# contribution.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public License,
+# v. 2.0. If a copy of the MPL was not distributed with this file, You can
+# obtain one at http://mozilla.org/MPL/2.0/.
+#
+# END HEADER
+
+from __future__ import division, print_function, absolute_import
+
+import math
+
+from hypothesis import given
+from tests.common.utils import fails
+from hypothesis.strategies import fractions
+from hypothesis.internal.compat import ceil, floor, integer_types
+
+
+@fails
```

Comment:
I'm not going to insist on this for this change, but I think it would make more sense to have positive tests that validate that ceil and floor have the properties we want. i.e. that `x <= ceil(x) <= x + 1` and `x - 1 <= floor(x) <= x` and then put the tests in `tests/nocover` rather than making it py2 specific.